lldb/test/API/functionalities/process_save_core_minidump/TestProcessSaveCoreMinidump.py

```diff
@@ -106,39 +108,39 @@ def test_save_linux_mini_dump(self):
 
             self.runCmd(base_command + " --style=modified-memory '%s'" % (core_dirty))
             self.assertTrue(os.path.isfile(core_dirty))
-            self.verify_core_file(
-                core_dirty, expected_pid, expected_modules, expected_threads, stacks_to_sp_map
-            )
+            # self.verify_core_file(
+            #     core_dirty, expected_pid, expected_modules, expected_threads, stacks_to_sp_map
+            # )
```

Comment:
We need to still test this. Was this intentionally commented out?

Reply:
Mistake on my part, file didn't finish saving before I added. (Resolving all of these).